Maple2.Database/Storage/Game/GameStorage.GameEvent.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Maple2.Model.Game.Event;
+
+namespace Maple2.Database.Storage;
+
+public partial class GameStorage {
+    public partial class Request {
+        public IEnumerable<GameEvent> GetEvents() {
```

Comment:
Probably fine to just return `IList<>` here.

```cs
return Context.GameEvent.Where(model => model.BeginTime <= DateTimeOffset.UtcNow && model.EndTime >= DateTimeOffset.UtcNow)
    .Select(model => {
        GameEvent result = model;
        result.EventInfo.Id = model.Id;
        result.EventInfo.Name = model.Name;
        return result;
    }).ToArray();
```

Reply:
I wasn't able to do this. I needed to make the EventInfo's id and name defined. Not sure if there's a better way.

Reply:
I don't really understand why but it's okay. I'm not really against your current implementation.